tmt/steps/report/junit.py

```diff
@@ -101,7 +300,30 @@ class ReportJUnitData(tmt.steps.report.ReportStepData):
         option='--file',
         metavar='PATH',
         help='Path to the file to store JUnit to.',
-        normalize=lambda key_address, raw_value, logger: Path(raw_value) if raw_value else None)
+        normalize=tmt.utils.normalize_path)
+
+    flavor: str = field(
+        default=DEFAULT_FLAVOR_NAME,
+        option='--flavor',
+        metavar='FLAVOR',
+        choices=[DEFAULT_FLAVOR_NAME, CUSTOM_FLAVOR_NAME],
+        help=f"Name of a JUnit flavor to generate. By default, the '{DEFAULT_FLAVOR_NAME}' flavor "
+        "is used.")
```

Comment:
A multiline string can be used, tmt will dedent it properly. Way better than spurious trailing whitespace :)

Reply:
Should be fixed now.